caffe2/core/operator.h

```diff
@@ -536,6 +533,10 @@ class CAFFE2_API OperatorBase : public Observable<OperatorBase> {
     return helper_;
   }
 
+  std::vector<at::Tensor> ivalue_outputs() && {
```

Comment:
This method is a bit shocking. I don't generally think of Operator as having move semantics...

Reply:
This is the only efficient way I found of returning the outputs. The issue is that an operator might set some outputs and leave others untouched. For this diff, I'm renaming the function to `move_newstyle_outputs` to make it more obvious but you're right, this is going to have to be changed once we want to keep operator instances around for multiple calls.